docs/content.md

```diff
@@ -28,6 +28,9 @@ This documentation aims to be a reference for content editors that want to make
 	* [Category Pages](#category-pages)
```

Comment:
Do we need to add something into the main guidance about downloading VS code and using it locally and then how you commit changes, push them e.t.c.?